core/src/main/java/org/eclipse/krazo/lifecycle/RequestLifecycle.java

```diff
@@ -43,6 +44,8 @@
 
     private boolean controllerExecuted = false;
 
+    private Method invokedMethod;
```

Comment:
Let's call this `controllerMethod` instead. I think that's a better fit.

Reply:
I'd name it `invokedControllerMethod`, as the name `controllerMethod` is already taken by the `Callable` in `aroundController`. But maybe we find a better name for the current `controllerMethod` field?

Reply:
Ah, I see. Maybe the `aroundController` method could look like this:

```
public Object aroundController(Method method, Callable<Object> invocation) .. {
```

I know that `method` is a bit generic, but maybe ok for this small method. Then we could call the field `controllerMethod` or `invokedControllerMethod`.

Reply:
Sounds good, I'll change it this way